packages/cli/build/main.ts

```diff
@@ -33,17 +34,26 @@ export default async function main(mode: 'dev' | 'build') {
         if (!confirm) return process.exit(0)
     }
 
-    const command = ['--mode', mode === 'dev' ? 'development' : 'production']
-    if (mode === 'dev') command.unshift('serve')
-    args.filter((x) => !x.startsWith('-')).forEach((target) => {
-        command.push('--env', target)
-        if (!knownTargets.includes(target)) {
+    // prettier-ignore
+    const command = [
+        'webpack',
+        '--mode',
+        mode === 'dev' ? 'development' : 'production',
+        mode === 'dev' ? 'serve' : undefined,
```

Comment:
Is `webpack --mode development serve` the same effect as `webpack serve --mode development`?